Sources/Basic/OutputByteStream.swift

```diff
@@ -405,3 +405,95 @@ public struct Format {
         return TransformedSeparatedListStreamable(items: items, transform: transform, separator: separator)
     }
 }
+
+/// Represents a stream which is backed to a file. Not for instantiating.
+public class FileOutputByteStream: OutputByteStream {
+
+    /// Closes the file flushing any buffered data.
+    public func close() {
+        // For subclasses to implement.
```

Comment:
It doesn't auto close on denit. Should it assert? because you might not want to close streams like stdout... however there should/is be only one public instance of stdoutStream.